packages/ffe-datepicker/less/calendar.less

```diff
@@ -157,3 +151,250 @@
+            width: 14%;
+            margin-right: 0.5 * @margin-between;
+        }
+
+        > div:nth-child(2) {
+            height: 100%;
+            flex-grow: 3;
+            margin: 0 0.5 * @margin-between;
+        }
+
+        > div:nth-child(3) {
+            height: 100%;
+            flex-grow: 1;
+            width: 14%;
+            margin-left: 0.5 * @margin-between;
+            > * {
+                width: 100%;
+                height: 100%;
+                margin: 0;
+            }
+        }
+    }
+
+    &-inner-3 {
+        position: absolute;
+        left: 9px;
+        right: 9px;
+        top: 9px;
+        bottom: 9px;
+        display: flex;
+        justify-content: space-between;
+
+        > div:nth-child(1) {
+            height: 100%;
+            width: 150px;
+        }
+
+        > div:nth-child(2) {
+            height: 100%;
+            width: 230px;
+        }
+
+        > div:nth-child(3) {
+            height: 100%;
+            width: 150px;
+            > * {
+                width: 100%;
+                height: 100%;
+                margin: 0;
+            }
+        }
+    }
+}
+
+.ffe-calendar__monthpicker {
+    background-color: @ffe-white;
+    border: 2px solid @ffe-grey-silver;
+    border-radius: 4px;
+    box-shadow: 0 1px 1px 0 @ffe-grey-cloud;
+    color: @ffe-black;
+    font-family: MuseoSans-500, arial, sans-serif;
+    height: 45px;
+    line-height: 20px;
+    transition: all @ffe-transition-duration @ffe-ease;
+    width: 100%;
+    position: relative;
+
+    @media (any-hover: hover) {
+        &-left:hover {
+            > div {
+                background-position: 50% 70%;
+            }
+        }
+    }
+
+    &-left {
+        position: absolute;
+        left: 0;
+        top: 0;
+        bottom: 0;
+        width: 45px;
+        cursor: pointer;
+
+        > div {
+            transition: all @ffe-transition-duration @ffe-ease;
+            position: relative;
+            margin: 0 auto;
+            height: 100%;
+            width: 30px;
+            transform-origin: 50% 50%;
+            background-position: 50% 80%;
+            transform: rotate(90deg);
+            background-repeat: no-repeat;
+            @bare-color: escape(@ffe-blue-azure);
+
+            background-image: url('data:image/svg+xml;charset=utf8,%3Csvg%20xmlns%3Asvg%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20version%3D%221.1%22%20x%3D%220%22%20y%3D%220%22%20viewBox%3D%220%200%201000%201000%22%20width%3D%2218%22%20height%3D%2218%22%20xml%3Aspace%3D%22preserve%22%3E%0A%20%20%20%20%3Cpath%20fill%3D%22@{bare-color}%22%20d%3D%22M111.1%20228.6c19.2%200%2038.4%207.3%2053%2022l336.7%20336.7%20334.5-334.5c29.3-29.3%2076.8-29.3%20106.1%200%2029.3%2029.3%2029.3%2076.8%200%20106.1L553.8%20746.3c-29.3%2029.3-76.8%2029.3-106.1%200L58.1%20356.6c-29.3-29.3-29.3-76.8%200-106.1C72.7%20235.9%2091.9%20228.6%20111.1%20228.6z%22%20%2F%3E%0A%3C%2Fsvg%3E%0A');
```

Comment:
What's this, and the one further down?
We have icons from ffe-icons(-react). If we want to look the change of something, like a chevron, it should only need to be changed in one place.